src/utils/fs.ts

```diff
@@ -30,3 +47,13 @@ export function getRandomHexString(length: number = 10): string {
     const buffer: Buffer = crypto.randomBytes(Math.ceil(length / 2));
     return buffer.toString('hex').slice(0, length);
 }
+
+export function isPathEqual(fsPath1: string, fsPath2: string): boolean {
+    const relativePath: string = path.relative(fsPath1, fsPath2);
+    return relativePath === '';
+}
+
+export function isSubPath(fsPath1: string, fsPath2: string): boolean {
+    const relativePath: string = path.relative(fsPath1, fsPath2);
+    return relativePath !== '' && !relativePath.startsWith('..');
```

Comment:
nit: seems like !== '' not necessary

Reply:
If I remove that check, then you get 'true' if the paths are the same.

It doesn't seem like `/example/path/` is a subfolder of `/example/path/`